python/pdo/submitter/submitter.py

```diff
@@ -24,12 +24,12 @@ class Submitter(object):
 
     def __init__(self, ledger_config, *args, **kwargs):
 
-        self.url = ledger_config.get('LedgerURL','http://localhost:8008')
+        self.url = ledger_config.get('LedgerURL','http://localhost:6600')
         self.pdo_signer = kwargs.get('pdo_signer', None) #PDO payload signer
```

Comment:
again... does this even work for CCF? doesn't it need to be the IP address?

Reply:
Sort of: what's the default IP? 
We don't know.
I think we can't use default values here. I'll just throw an error.

Reply:
Done

Reply:
In theory... you cannot get this far without setting PDO_LEDGER_URL. This is a bigger problem. We force them to be set, then check everywhere with all this extra complexity. Universally these should just be errors.